### PR TITLE
MINOR: Document JBOD limitation during migration

### PR DESCRIPTION
**Pull request by soarez**

### CODE REVIEW

File: docs/ops.html [hunk 1]
```diff
@@ -3823,6 +3823,7 @@ <h3>Limitations</h3>
       version</i> (also known as the <i>inter.broker.protocol.version</i>.) Please do not attempt to do this during
       a migration, or you may break the cluster.</li>
     <li>After the migration has been finalized, it is not possible to revert back to ZooKeeper mode.</li>
+    <li>During the migration, if a ZK broker is running with multiple log directories, any directory failure will cause the broker to shutdown.</li>
```
Comment (chia7712) [inline docs/ops.html]: If I'm a user, I'd like to know - 
1) the jira link about "why" 
2) the workaround about "how"
WDTY?

Comment (showuon) [reply docs/ops.html]: Also, I think that's the current status (v3.7.0), right? Will we fix that in the future release?

Comment (soarez) [reply docs/ops.html]: @chia7712 great points. Updated. PTAL

@showuon No, the current plan/intention is to have that as simplification during the migration. See https://issues.apache.org/jira/browse/KAFKA-16431. Please let me know if you have any concerns with this approach. cc @cmccabe @pprovenzano

File: docs/ops.html [hunk 2]
```diff
@@ -3823,6 +3823,7 @@ <h3>Limitations</h3>
       version</i> (also known as the <i>inter.broker.protocol.version</i>.) Please do not attempt to do this during
       a migration, or you may break the cluster.</li>
     <li>After the migration has been finalized, it is not possible to revert back to ZooKeeper mode.</li>
+    <li>During the migration, if a ZK broker is running with multiple log directories, any directory failure will cause the broker to shutdown. Brokers with broken log directories will only be able to migrate to KRaft once the directories are repaired. For further details refer to <a href="https://issues.apache.org/jira/browse/KAFKA-16431">KAFKA-16431</a>.</li>
```
Comment (chia7712) [inline docs/ops.html]: Could we split it into multiple lines?

**Review by chia7712 — COMMENTED** [review]
@soarez the docs LGTM except for one small comment :)

**Review by chia7712 — APPROVED** [review]
LGTM

**Review by showuon — APPROVED** [review]
LGTM!

**Pull request merged**